### ISSUE TITLE
Create separate type of deck activities for card assignments email notification

### ISSUE THREAD

**Issue by klemens-u** [nextcloud/deck#1238]
I want to have an option to receive notifications per email only when someone assigns a card to me. In no other cases.

That would be very nice to receive notification in this important case, but reduce "spam" for other less important messages.
Furthermore it simulates the behaviour of similar systems.

Thanks for the app so far, it has great potential!

**Comment by stephen-cidtek** [comment]
Expanded notifications for this app would be greatly appreciated, especially on card assignment as mentioned above. 

Can bounties be placed on requested features?

**Comment by klemens-u** [comment]
Still active Mr. Stalebot!

**Comment by Mangleuz** [comment]
I agree with the need for this and the statement above: "Furthermore it simulates the behaviour of similar systems".
Without this feature the many unnecessary emails becomes very annoying.

**Comment by agattorno** [comment]
I agree. This is very important. NC 21

**Comment by stefan-niedermann** [comment]
@agattorno Please use **[GitHub reactions](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/)** 👍 to show that you are affected by the same issue. Comments without any new information will notify every subscribed user and make it harder for developers to understand and follow an issue. Note that the Deck app has an [independent release cycle](https://github.com/nextcloud/deck/releases), so the Nextcloud version does not matter.
